packages/components/src/components/send-email-template/send-email-template.tsx

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import classNames from 'classnames';
```

Comment:
I think we can remove classNames from the file

Reply:
Fixed. Thanks for highlighting 🙇